modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc

```diff
@@ -21,7 +28,7 @@ The following sections cover:
 This document describes the steps required to extend the {prod-short} telemetry system to connect to a custom back-end:
 
 . Creating a server process that receives events
-. Extending {prod-short} libraries to create a back-end that send events to the server
+. Extending {prod-short} libraries to create a back-end that sends events to the server
```

Comment:
@dkwon17, could you please fix the nouns (nouns only) to `backend`.
Checking the IBM Style guide, and it has a rule for this:
![image](https://user-images.githubusercontent.com/78564304/159701862-525f18a8-13fc-4792-8710-4bda1b9655ca.png)